pkg/coinbase/balance.go

```diff
@@ -38,9 +38,15 @@ func newBalanceFromModel(model *client.Balance) (*Balance, error) {
 	if err != nil {
 		return nil, err
 	}
-	amount, ok := new(big.Int).SetString(model.Amount, 10)
-	if !ok {
-		return nil, fmt.Errorf("failed to parse amount: %s", model.Amount)
+
+	amount := big.NewInt(0)
+
+	if model.GetAmount() != "" {
```

Comment:
Is this because we are sometimes returning an empty value instead of a 0 value for a balance check?

Reply:
Exactly 

The following was set to "" instead of '0'
```
 unbondedBalance: 'Balance { amount: '0' asset: 'Asset { networkId: '' assetId: '' contractAddress: '' decimals: '0' }' }
```

Reply:
AH makes sense, for solana its not a 0 value, but an nil one? If were just having the SDKs make nil -> 0, should we just return 0 from the API?

Reply:
I think making it nil on the api would be better